server/runtime/src/main/server/bin/report-osx.sh

```diff
@@ -32,13 +32,13 @@ uname -a > "$TMPDIR/uname"
 df -h > "$TMPDIR/df"
 lsof -itcp -P -n | grep "LISTEN"  > "$TMPDIR/ss-tcp"
 lsof -iudp -P -n > "$TMPDIR/ss-udp"
-lsof / > "$TMPDIR/lsof" 2> /dev/null
+lsof -b / > "$TMPDIR/lsof" 2> /dev/null
```

Comment:
There's a couple lsof calls above, should the have the option, too?

Reply:
I'll ask a mac owner to test that